src/mnt/fuse_pure.rs

```diff
@@ -23,9 +23,11 @@ use std::process::{Command, Stdio};
 use std::sync::Arc;
 use std::{mem, ptr};
 
-const FUSERMOUNT_BIN: &str = "fusermount";
-const FUSERMOUNT3_BIN: &str = "fusermount3";
-const FUSERMOUNT_COMM_ENV: &str = "_FUSE_COMMFD";
+#[cfg(target_os = "linux")]
```

Comment:
Because of all the conditional compilation, there's now a fair bit of dead code/unused functions warnings in this file. I wonder if it's viable to just silence them with `#![allow(dead_code)]` and similar?